.github/workflows/pull-request.yml

```diff
@@ -19,3 +40,5 @@ jobs:
             PR is now waiting for a maintainer to run the acceptance tests. This PR will only perform build and linting.
             **Note for the maintainer:** To run the acceptance tests, please comment */run-acceptance-tests* on the PR
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+Expect(err):
```

Comment:
This looks like copypasta (or I don't understand github actions as well as I thought)

Reply:
😱 Yeah that doesn't make any sense.